appdatabase/migrations/sql/1719583610_add_connector_dapps.up.sql

```diff
@@ -0,0 +1,8 @@
+-- connector_dapps table keeps track of connected dApps to provide a link to their individual sessions
+-- should be aligned with wallet_connect_dapps table
+
+CREATE TABLE IF NOT EXISTS connector_dapps (
```

Comment:
i'm not sure if `appdatabase/migrations` is a good place for connector migrations